aws-samples/eks-blueprints-for-proton
# feat: Update modules used to latest versions

*Issue #, if available:*
- N/A: Slack request

*Description of changes:*
- Update modules to use latest versions
- Use S3 bucket module with baked in settings for security and versioning of remote state
- Add in local tags; this would ideally be replaced by a users own set of tags or tag module but at least we are showing tags and can relate those resources back to this project
- Update files to follow Terraform conventions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
